src/api/fetchProductSummary.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+import type { Product } from '@/types';
+
+const API_BASE = import.meta.env.VITE_API_BASE_URL;
+
+export async function fetchProductSummary(productId: string): Promise<Product> {
+  const { data } = await axios.get(`${API_BASE}/api/products/${productId}`);
+  return data.data;
+}
```

Comment:
지금은 `fetchProductSummary` 함수의 리턴값에 타입을 지정해주고 계셔서,
`data` 변수는 현재 `any` 타입을 가지고 있는데요!

`axios`의 `get` 메서드에 타입스크립트 제네릭을 적용하는 방식을 사용하면 data에 타입을 지정하고,
함수의 리턴타입을 제거할 수 있습니다 ㅎㅎ

```suggestion
export async function fetchProductSummary(productId: string) {
  const { data } = await axios.get<{data : Product}>(`${API_BASE}/api/products/${productId}`);
  return data.data;
}

```